src/main/java/com/cs56fitnessapp/models/Day.java

```diff
@@ -1,13 +1,33 @@
 package com.cs56fitnessapp.models;
 
 /**
- * @author Ksenia Koldaeva
- * Created: 9/17/17
- * Last Updated: 10/1/17
+ * @author Saori Shigehisa
+ * Created: 10/6/17
+ * Last Updated: 10/7/17
  */
+import java.util.ArrayList;
+import java.util.Date;
 
 public class Day implements ActivityFacts, DietFacts {
 
+    /** Date */
+    private Date date;
+
+    /** User */
+    private User user;
+
+    /** calorieGoal */
+    private double calorieGoal;
+
+    /** WorkoutList */
+    private ArrayList < Workout > workoutList;
+
+    /** MealList */
+    private ArrayList < Meal > mealList;
+
+    public boolean isToday() { return false; }
```

Comment:
You have to actually implement `isToday()`. E.g it should check the `date` field and return true if it's today's date.

Reply:
done

Reply:
made if the date is matched, it is true.